tests/test_indices.py

```diff
@@ -672,6 +704,13 @@ def test_standardized_precipitation_index(
             and Version(__numpy_version__) < Version("2.0.0")
         ):
             pytest.skip("Skipping SPI/ML/D on older numpy")
+
+        # change `dist` to a lmoments3 object if needed
+        if method == "PWM":
+            lmom = pytest.importorskip("lmoments3.distr")
+            scipy2lmom = {"gamma": "gam"}
+            dist = getattr(lmom, scipy2lmom[dist])
```

Comment:
@Zeitsperre are we sure that any `*-lmoments3` tox test suite is running here ? 

@coxipi Because I think the test should fail : `dist` is passed as a `lmoments3` object in the test but in the SPI and SPEI functions there are tests like `dist in dist_methods`, which would fail. I'm not sure how to do it, but the function/indicator must now be able to accept scipy/lmoments3 distribution objects if the `method=='PWM'` is to be enabled!

Reply:
Ah you're right! in the tests I have called the low-level function, so it's really `params` that counts in the SPI call. The default distribution that is sent in the warning message does not matter.

Reply:
Confirming here that tests using `lmoments3` are being run.